app/scripts/services/aggregatedLoggingService.js

```diff
@@ -0,0 +1,47 @@
+'use strict';
+
+angular.module("openshiftConsole")
+  .factory("AggregatedLoggingService", function($q, Logger, DataService) {
+
+    // cache previous request
+    var userAllowed;
+
+    // Create SelfSubjectAccessReview request againt authorization API to check whether
+    // current user can view pods/log from 'default' project.
+    // Users who can view such logs are 'operations' users
+    var isOperationsUser = function() {
+      var authPromise = null;
+
+      if (userAllowed === undefined) {
```

Comment:
I'd switch this to an early return if we have a cached value. It's easier to reason about.

```js
if (userAllowed !== undefined) {
  return $q.when(userAllowed);
}

var ssar = { ... };
return DataService.create(...).then(...);
```